src/food_journal.py

```diff
@@ -0,0 +1,170 @@
+from datetime import datetime, timedelta
+import json
+import psycopg2
+
+from postgres import start_connection, close_connection
+
+BREAKFAST = "Breakfast"
+LUNCH = "Lunch"
+DINNER = "Dinner"
+SNACK = "Snack"
+MIDNIGHTSNACK = "Midnight Snack"
```

Comment:
Python has support for enums, check out https://docs.python.org/3/library/enum.html